lib/matplotlib/table.py

```diff
@@ -655,7 +656,7 @@ def table(ax,
           cellLoc='right', colWidths=None,
           rowLabels=None, rowColours=None, rowLoc='left',
           colLabels=None, colColours=None, colLoc='center',
-          loc='bottom', bbox=None, edges='closed',
+          loc='bottom', bbox=None, edges='closed', df=None,
```

Comment:
```suggestion
          loc='bottom', bbox=None, edges='closed',
```
This isn't used. Looks like this came in accidentally.

Reply:
You are right. I fixed this in the next commit.